integration_test/tests/common/mod.rs

```diff
@@ -33,3 +34,45 @@ pub fn generate_default_connection_str() -> String {
 
     connection_string
 }
+
+#[allow(dead_code)]
```

Comment:
```suggestion
#[test]
```
would `#[test]` be more appropriate than `#[allow(dead_code)]` since this could isn't "dead" but instead is used in codepaths guarded by `#[test]`? or maybe i'm losing the thread of how this function is being used!

Reply:
ah, this one was a mystery to me.
In lib.rs, this is flagged as code used for tests.
```
#[path = "../tests/common/mod.rs"]
#[cfg(test)]
mod common;
```
 It is also used similarly to generate_default_connection_str.
 So, I don't know what I missed, but it seemed more like a false positive that Clippy was flagging so I put the tag to allow dead code.
 If I missed something and I can clean this up, let me know.
 #[test] does not work because this is not a test, but a helper function used by multiple tests.

Reply:
Oh right right, I meant `#[cfg(test)]` 🤦‍♀️  But if the entire module is already marked with that, then I'm not sure adding it at the function level will do anything anyway!